tests/test_pandascore.py

```
@@ -181,6 +181,20 @@ def test_extract_match_data_valid(parser):
         assert result["team2_id"] == 200
         assert result["best_of"] == 3
 
+    @staticmethod
+    def test_extract_match_data_normalizes_lol_payload_slug(parser):
```

Ruff (0.15.5)

[warning] Missing return type annotation for staticmethod `test_extract_match_data_normalizes_lol_payload_slug`

Add return type annotation: `None`

(ANN205)